prototype-spec.md

```diff
@@ -0,0 +1,253 @@
+# Notary v2 Specification
+
```

Comment:
This specification describes the file formats, queries, and processes required to use TUF in registries.  It requires that metadata for each of the TUF roles--root, snapshot, timestamp, and targets--be  uploaded to a registry and capable of being queried by users.